Dockerfile

```diff
@@ -20,6 +20,8 @@ ARG MIXPANEL_KEY
 ARG HOSTNAME_PARTS
 ARG SWAP_CP
 ARG VIEWER_PAGE_SUBDOMAIN
+ARG BASE_GITHUB_PATH
```

Comment:
I'm good with this change, but I'd like the arg name to be a bit more clear. Maybe something like GITHUB_JS_PATH and CSN_JS_PATH? Just to indicate a little better that it's for the Javascript files.

Reply:
Sure I can change that

Reply:
done